packages/api-demo/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/notifications/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
I tried that first actually, and it didn't work.

The js file is served as the url `/scripts/containerjs-api-bundle` (without `.js`).
The template file is expected relative to the screen, so `/notifications/notification.html`

As discussed this morning, this is perhaps an area where the API needs looking at anyway, so I'd hope this would change to something better.